sdk/ml/azure-ai-ml/azure/ai/ml/entities/_compute/unsupported_compute.py

```diff
@@ -25,13 +25,18 @@ def __init__(
     @classmethod
     def _load_from_rest(cls, rest_obj: ComputeResource) -> "UnsupportedCompute":
         prop = rest_obj.properties
+        if hasattr(rest_obj, "tags"):
+            # DataFactory object has no tags
```

Comment:
add the bug link here: [Bug 2294131](https://msdata.visualstudio.com/Vienna/_workitems/edit/2294131): [sdkv2+cliv1.5] Job submission failure 'DataFactory' object has no attribute 'tags'